command/commands.go

```diff
@@ -145,13 +145,10 @@ var (
 	}
 
 	physicalBackends = map[string]physical.Factory{
-		"file_transactional":     physFile.NewTransactionalFileBackend,
-		"file":                   physFile.NewFileBackend,
-		"inmem_ha":               physInmem.NewInmemHA,
-		"inmem_transactional_ha": physInmem.NewTransactionalInmemHA,
-		"inmem_transactional":    physInmem.NewTransactionalInmem,
```

Comment:
Probably need to add this back, now that I added a transactional InMem data store to compare the raft one against.

Reply:
I think I'll leave it off: we probably want inmem to be transactional by default.